java/src/jmri/jmrix/can/CanSystemConnectionMemo.java

```diff
@@ -104,7 +104,12 @@ public boolean provides(Class<?> type) {
             return mgr.isAddressedModePossible();
         }
         if (type.equals(jmri.ConsistManager.class)) { // until a CAN ConsistManager is implemented, use Internal
-            return false;
+            if (_protocol.equals(ConfigurationManager.SPROGCBUS)) {
+                // SPROG CBUS DOES have a consist manager but we'll do things correctly and check with the manager
+                return manager.provides(type);
+            } else {
+                return false;
```

Comment:
As I read the code, this will always return false if the protocol isn't ConfigurationManager.SPROGCBUS.
Should it also return true for jmri.jmrix.can.ConfigurationManager.MERGCBUS ?

Perhaps remove the whole type.equals(jmri.ConsistManager.class) block from ln 106-112, and let the actual ConfigurationManager decide, ln 114.

CbusConfigurationManager.provides(jmri.ConsistManager.class) is changed to true so all good there :-)
The other ConfigurationManagers should be returning false by default for an unsupported class.

Reply:
The consist tool was being provided for MERG connections, just not the way I thought. I was a bit concerned by the original hard coded return false but you are correct that the other ConfigurationManagers all return false by default, so I have removed that block of code.